### ISSUE TITLE
iOS - Arrows from an Opened Link on the "About" Screen - Have Poor Color Contrast

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#151]
I noticed the there are two arrows, for I assume "Previous" and "Next" links, and they have poor color contrast. If you made the arrows white ( #ffffff ) on the dark gray background ( #494949 ) like the "Done" button text, they would pass.

Plus, when reading with VoiceOver, they both read "Unpronounceable". I am assuming they should be "Previous" and "Next" links.

**NOTE**: This issue is likely the same for Android and web too.

Please see the attached screenshot.

![ilab - comply chain - ios - about screen - color contrast issue with arrows at bottom of screen](https://user-images.githubusercontent.com/10423487/91578434-2b1c8e00-e918-11ea-85d9-63eb44e30c8a.png)

**Comment by jfc3-dol** [comment]
The link at top of screen after opening an external page on the About screen do not have enough color contrast. If you change the link text color to black ( #000000 ) or a charcoal gray ( #3d3d3d ) it will pass. 

 I assume this could be an issues on other pages too.

Please see the attached screenshot.

![ilab comply chain - about screen - Links at top of screen do not have enoguh colro contrast](https://user-images.githubusercontent.com/10423487/116723035-157ecd00-a9ad-11eb-83dc-0e529edc1af6.png)

**Comment by jfc3-dol** [comment]
There is still a color contrast issue for the "Previous" and "Next" buttons pointed out on August 28, 2002.

**Comment by jfc3-dol** [comment]
@pbhatt17

The color contrast ratio is 2.9:1 and needs to be 4.5:1 or higher to pass. If you can not make the highlighted arrows' color lighter, such as white like the text. Then change the background to black.

Please see the attached screenshot.

<img width="375" alt="ilab - comply chain - about page - after opening link the previous and next arrows do not have enough color contrast" src="https://user-images.githubusercontent.com/10423487/129956490-7718031c-9fd1-493a-badf-0f0eb521b953.png">

**Comment by jfc3-dol** [comment]
@pbhatt17 

The "Previous" and "Next" arrows ( #4A4A4A ) on the dark gray background ( #1C1C1C ) have a color contrast ratio of 1.9:1 and need to be 4.5:1 or higher to pass.

Either make the arrows white or the same shade of blue of the "Done" text to pass.
